src/database/seeders/20220323100028-demo-accomodation.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable strict */
+/* eslint-disable import/no-import-module-exports */
+
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    try {
+      await queryInterface.bulkInsert(
+        'accomodations',
+        [
+          {
+            id: 3,
```

Comment:
accommodation id is auto increment based

Reply:
This particular Id is needed in add-trip seeders so the auto-incremented value can't be used